### ISSUE TITLE
Option to not crop image in thumbnail?

### ISSUE THREAD

**Issue by lambdafu** [liwenyip/hugo-easy-gallery#29]
For some type of images (technical diagrams, screenshots), it seems better to not crop the thumbnail into a square, but to display the thumbnail with a border (so the whole image is visible). Is there an easy way to do this?

**Comment by garchymede** [comment]
yes, it is possible and quite easy to do by modifying the file named _**hugo-easy-gallery.css**_ : 

> .gallery .img {
>     position: absolute;
>     left: 0;
>     right: 0;
>     top: 0;
>     bottom: 0;
>     **background-size: contain;**
